woocommerce/payment-gateway/External_Checkout/Admin.php

```diff
@@ -390,13 +390,8 @@ protected function get_configuration_errors() {
 		if ( ! empty( $accepted_currencies ) && ! in_array( get_woocommerce_currency(), $accepted_currencies, true ) ) {
 
 			$errors[] = sprintf(
-				/* translators: Placeholders: %1$s - plugin name, %2$s - a currency/comma-separated list of currencies, %3$s - <a> tag, %4$s - </a> tag, %5$s - external checkout label */
-				_n(
-					'Accepts payment in %1$s only. %2$sConfigure%3$s WooCommerce to accept %1$s to enable %4$s.',
-					'Accepts payment in one of %1$s only. %2$sConfigure%3$s WooCommerce to accept one of %1$s to enable %4$s.',
-					count( $accepted_currencies ),
-					'woocommerce-plugin-framework'
-				),
+				/* translators: Context: Error message displayed to merchants if their store currency is not in one of the accepted currencies by the payment gateway they intend to enable. Placeholders: %1$s - a currency or comma-separated list of currencies, %2$s - opening HTML <a> tag, %3$s - closing HTML </a> tag, %4$s - payment method title/label */
```

Comment:
the problem here was that the translators placeholder was placed above the line where the translatable text was, maybe it wasn't parsed correctly in gettext, so in this occurrence, and others, I have rearranged it

Reply:
Yup, that was probably it